packages/css/src/index.js

```diff
@@ -35,6 +37,8 @@ const aliases = {
 const multiples = {
   marginX: ['marginLeft', 'marginRight'],
   marginY: ['marginTop', 'marginBottom'],
+  marginXBottom: ['marginLeft', 'marginRight', 'marginBottom'],
+  marginXTop: ['marginLeft', 'marginRight', 'marginBottom'],
```

Comment:
should the third value in this array be `marginTop`?

Reply:
Yes. Thank you.

Reply:
Yes. Thank you.

Reply:
Yes. Thank you.